src/tree-node/index.test.js

```diff
@@ -94,3 +94,31 @@ test('should render data attributes', t => {
   t.is(wrapper.prop('data-first'), 'john')
   t.is(wrapper.prop('data-last'), 'smith')
 })
+
+test('should set aria-selected to true for checked nodes', t => {
+  const node = {
+    _id: '0-0-0',
+    _parent: '0-0',
+    label: 'item0-0-0',
+    value: 'value0-0-0',
+    className: 'cn0-0-0',
+    checked: true
+  }
+
+  const wrapper = mount(<TreeNode {...node} />)
+  t.is(wrapper.prop('aria-selected', true))
```

Comment:
Can we generate a snapshot for this instead of checking specific props? Nothing wrong with this in general, but just to be consistent with other tests in here.

Reply:
Done! I had some problems with the tree test though so I removed it. I see both the other snapshots test there are skipped so they might all be problematic?

Reply:
The skipped ones have a pending issue open with ava - they cause the tests to hang for some unknown reason. 

When your changes result in updated snapshots, the tests will fail. You then need to verify whether the changes in the rendered HTML are expected or unintentional. If expected, you can update the snapshots by running `yarn test --update-snapshots` (see [more](https://github.com/avajs/ava/blob/master/docs/04-snapshot-testing.md)) and commit and push those snapshots. 

I see that the CI build has some failed snapshots tests. Those would pass once you checkin updated snapshots. Let me know if you have more questions.